figma-tokens.json

```diff
@@ -599,20 +567,142 @@
       },
       "md": {
         "size": {
-          "value": "{size.element.lg}",
+          "value": "{size.12}",
           "type": "sizing"
         }
       },
       "shape": {
         "circle": {
           "value": "{borderRadii.full}",
-          "type": "borderRadii"
+          "type": "borderRadius"
         },
         "rounded": {
-          "value": "{element.borderRadii.md}",
-          "type": "borderRadii"
+          "value": "{borderRadii.3}",
+          "type": "borderRadius"
         }
       }
     }
-  }
-}
+  },
+  "component.button": {
```

Comment:
would you please follow the component structure where we wrap all tokens in component name
<img width="451" alt="Screen Shot 2022-08-03 at 09 12 33" src="https://user-images.githubusercontent.com/52660542/182546836-46a617f1-2239-4279-8695-89ef9d120634.png">